jujugui/static/gui/src/app/app.js

```diff
@@ -2803,6 +2806,7 @@ YUI.add('juju-gui', function(Y) {
         console.log('switching models is not supported in sandbox');
       }
       console.log('switching model connection');
+      this.env.set('available', false);
```

Comment:
Why setting this as a YUI attribute? Do we plan to have subscribers for "available" changes? Also, why is this an attribute of the model connection? As the logic around model availability is only relevant at the app level, what do you think about making this an app._modelIsAvailable attribute?